spec/lib/gettext_setup_spec.rb

```diff
@@ -51,4 +51,19 @@
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
     end
   end
+  context 'multiple locales' do
+    # locales/ loads the de locale and alt_locales/ loads the jp locale
+    before(:all) do
+      GettextSetup.initialize(File::join(File::dirname(File::dirname(__FILE__)), 'fixtures', 'alt_locales'))
+    end
+    it 'can aggregate locales across projects' do
+      expect(FastGettext.default_available_locales).to include('en','de','jp')
+    end
+    it 'can switch to loaded locale' do
+      FastGettext.locale = GettextSetup.negotiate_locale('de')
```

Comment:
Did you want to do an expectation prior to changing it i.e. is it currently NOT de before moving

or does the next expectation (moving to JP) already cover this?